### PR TITLE
Task4608 refactor vuetify input docs

### PR DESCRIPTION
**Pull request by Liliya-web**

### CODE REVIEW

File: source/includes/documentation/_vue_js_elements.md [hunk 1]
```diff
@@ -2964,3 +2964,128 @@ Available methods in Java JDI Light:
...
+<style>
+  #input-usage .v-input__prepend-outer,
+  #input-usage .v-input__append-outer,
+  #input-usage .v-input__slot,
+  #input-usage .v-messages {
+    border: 1px dashed rgba(0,0,0, .4);
+  }
+</style>
+```
+
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:3034]: Mayabe just a smaller piece from site? ^^
`<template>
  <v-container
    id="input-usage"
    fluid
  >
    <v-row>
      <v-col cols="12">
        <v-input
          :messages="['Messages']"
          append-icon="mdi-close"
          prepend-icon="mdi-phone"
        >
          Default Slot
        </v-input>
      </v-col>
    </v-row>
  </v-container>
</template>`

**Review by MayaElf — APPROVED** [review]

File: source/includes/documentation/_vue_js_elements.md [hunk 2]
```diff
@@ -2964,3 +2964,128 @@ Available methods in Java JDI Light:
...
+<style>
+  #input-usage .v-input__prepend-outer,
+  #input-usage .v-input__append-outer,
+  #input-usage .v-input__slot,
+  #input-usage .v-messages {
+    border: 1px dashed rgba(0,0,0, .4);
+  }
+</style>
+```
```
Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:3033]: Mayabe just a smaller piece from site? ^^
`<template>
  <v-container
    id="input-usage"
    fluid
  >
    <v-row>
      <v-col cols="12">
        <v-input
          :messages="['Messages']"
          append-icon="mdi-close"
          prepend-icon="mdi-phone"
        >
          Default Slot
        </v-input>
      </v-col>
    </v-row>
  </v-container>
</template>`

**Review by MayaElf — CHANGES_REQUESTED** [review]

File: source/includes/documentation/_vue_js_elements.md (hunk 2 above)
Comment (Liliya-web) [reply source/includes/documentation/_vue_js_elements.md:3033]: I added the same code as on the picture, is something wrong with it?

Comment (MayaElf) [inline source/includes/documentation/_vue_js_elements.md:3033]: Please, change according to Vladimir's comments:
in a documentation near highlighted comment please also emphasize a Selenium locator findby 
image

**Review by MayaElf — CHANGES_REQUESTED** [review]

**Review by MayaElf — APPROVED** [review]

**Review by vklonin — APPROVED** [review]

**Pull request merged**